pkg/core/bootstrap/bootstrap.go

```diff
@@ -71,9 +72,24 @@ func onStartup(runtime core_runtime.Runtime) error {
 	if err := createDefaultMesh(runtime); err != nil {
 		return err
 	}
+	if err := createPrivateKeyForInitialTokens(runtime); err != nil {
+		return err
+	}
 	return startReporter(runtime)
 }
 
+func createPrivateKeyForInitialTokens(runtime core_runtime.Runtime) error {
```

Comment:
Let's call it `createDefaultTokenIssuer`

Reply:
Good point that the name was old, but changed to `createDefaultSigningKey`, this functions creates a signing key for token issuer, not token issuer itself.